### PR TITLE
Add short descriptions to contributor docs

### PR DESCRIPTION
**Pull request by davepagurek**
This adds a short description to the top of each contributor doc within an HTML comment. These won't show up publicly on GitHub, but will be parsed out by the new website infrastructure and will be used on the contributor docs listing.

### CODE REVIEW

File: contributor_docs/access.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+<!-- What our commitment to access means for contributors and users of the library. -->
```
Comment (Qianqianye) [inline contributor_docs/access.md:1]: Hi @natdecker, please take a look at the one-line description of the access statement and share any feedback if you might have. Thanks!

**Comment by Qianqianye** [comment]
Thanks @davepagurek. While we are adding one-line description to the contributor docs, we need to archive or remove some of the contributor docs. For example, [organization.md](https://github.com/processing/p5.js/blob/main/contributor_docs/organization.md) is merged to [Steward Guidelines](https://github.com/processing/p5.js/blob/main/contributor_docs/steward_guidelines.md). I can work with @limzykenneth to organize of some 'older' contributor docs.

**Comment by Qianqianye** [comment]
Hi @davepagurek, could you rebase this PR on #6906? Thanks!

**Comment by davepagurek** [comment]
All rebased now!

**Pull request merged**